feil-behandler/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/feilbehandler/river/FeilLytter.kt

```diff
@@ -108,8 +108,9 @@ class FeilLytter(rapidsConnection: RapidsConnection, private val repository: Bak
             return false
         }
         val behovFraMelding = fail.utloesendeMelding.toMap()[Key.BEHOV]?.fromJson(BehovType.serializer())
-        val skalHaandteres = behovSomHaandteres.contains(behovFraMelding)
-        sikkerLogger.info("Behov: $behovFraMelding skal håndteres: $skalHaandteres")
+        val harRetry = fail.utloesendeMelding.toMap()[Key.RETRY]?.fromJson(String.serializer())
+        val skalHaandteres = harRetry != null || behovSomHaandteres.contains(behovFraMelding)
```

Comment:
Hvis du stoler på at `Key.RETRY` har verdi så kan du erstatte `harRetry != null` med `Key.RETRY in fail.utloesendeMelding.toMap()`.